framework/src/controller/controller.js

```diff
@@ -106,6 +108,18 @@ class Controller {
 		await fs.writeFile(pidPath, process.pid);
 	}
 
+	/**
+	 * Initiate application state
+	 *
+	 * @async
+	 */
+	async _initState() {
+		this.applicationState = new ApplicationState({
+			initialState: this.config.initialState,
```

Comment:
do we need to change the config?
I think it's better to keep it as it is and compute the initial state from existing config?
Also `initialState` doesn't mean much for the user, so original `system` was better for the name?

Reply:
I don't get the idea. We pass the part of the configuration used to initiate the state. That's why the name is as it is.

Reply:
i think the main point is `initialState` is not really clear to user.
Exposed variable should be descriptive what exactly is